mlflow/sklearn/__init__.py

```diff
@@ -10,1900 +10,20 @@
     NOTE: The `mlflow.pyfunc` flavor is only added for scikit-learn models that define `predict()`,
     since `predict()` is required for pyfunc model inference.
 """
-import inspect
-import functools
-import os
-import logging
-import numpy as np
-import pickle
-import yaml
-import weakref
-from collections import defaultdict, OrderedDict
-from copy import deepcopy
-from packaging.version import Version
-from typing import Any, Dict, Optional
-
-import mlflow
-from mlflow import pyfunc
-from mlflow.data.code_dataset_source import CodeDatasetSource
-from mlflow.data.numpy_dataset import from_numpy
-from mlflow.data.pandas_dataset import from_pandas
-from mlflow.entities.dataset_input import DatasetInput
-from mlflow.entities.input_tag import InputTag
-from mlflow.tracking.client import MlflowClient
-from mlflow.exceptions import MlflowException
-from mlflow.models import Model, ModelInputExample, ModelSignature
-from mlflow.models.model import MLMODEL_FILE_NAME
-from mlflow.models.signature import _infer_signature_from_input_example
-from mlflow.models.utils import _save_example
-from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE, INTERNAL_ERROR
-from mlflow.tracking.artifact_utils import _download_artifact_from_uri
-from mlflow.utils import _inspect_original_var_name
-from mlflow.utils.autologging_utils import get_instance_method_first_arg_value
-from mlflow.utils.environment import (
-    _mlflow_conda_env,
-    _validate_env_arguments,
-    _process_pip_requirements,
-    _process_conda_env,
-    _CONDA_ENV_FILE_NAME,
-    _REQUIREMENTS_FILE_NAME,
-    _CONSTRAINTS_FILE_NAME,
-    _PYTHON_ENV_FILE_NAME,
-    _PythonEnv,
-)
-from mlflow.utils import gorilla
-from mlflow.utils.requirements_utils import _get_pinned_requirement
-from mlflow.utils.file_utils import write_to
-from mlflow.utils.docstring_utils import format_docstring, LOG_MODEL_PARAM_DOCS
-from mlflow.utils.mlflow_tags import (
-    MLFLOW_AUTOLOGGING,
-    MLFLOW_DATASET_CONTEXT,
-)
-from mlflow.utils.model_utils import (
-    _get_flavor_configuration,
-    _validate_and_copy_code_paths,
-    _add_code_from_conf_to_system_path,
-    _validate_and_prepare_target_save_path,
+from ._sklearn import (
```

Comment:
Maybe absolute path over relative path? https://google.github.io/styleguide/pyguide.html#22-imports
```
from mlflow.sklearn.sklearn import ...
```

I think we can remove the leading underscore `_` and use the name `sklearn`.